.github/workflows/release-azure-pipelines.yml

```diff
@@ -119,6 +119,8 @@ extends:
           # 1ES PT requires explicit build task for Roslyn analysis. Auto-injected Roslyn task will use build logs from this build.
           - task: DotNetCoreCLI@2
             displayName: Build projects
+            env:
+              ADO_TOKEN: $(System.AccessToken)

```

Comment:
Is there a reason this is needed now and wasn't before?

Reply:
good question,  I am not 100% sure yet why it was not needed before.. maybe its because of some version update in csproj file?

Without that, release pipeline would fail at Build Projects step with errors like:
"##[error]src\AdoPat.Test\AdoPat.Test.csproj(0,0): Error NU1900: Warning As Error: Error occurred while getting package vulnerability data: Unable to load the service index for source https://office.pkgs.visualstudio.com/DefaultCollection/_packaging/Office/nuget/v3/index.json."
https://office.visualstudio.com/OE/_build/results?buildId=32364044&view=results